R/CITEseq.R

```diff
@@ -1,18 +1,145 @@
 .cord_blood <- function(ess_list)
 {
-    names(ess_list$experiments) <-
-        gsub("_Counts", "", names(ess_list$experiments))
-    MultiAssayExperiment::MultiAssayExperiment(
-        experiments=(ess_list$experiments)
-    )
+    names(ess_list$experiments) <- gsub("_Counts", "", names(ess_list$experiments))
+    mae <- MultiAssayExperiment::MultiAssayExperiment(experiments=(ess_list$experiments))
+    coldat <- sampleMap(mae)[,-c(1:2), drop=FALSE]
+    rownames(coldat) <- coldat[,1]
+    colnames(coldat) <- c("sampleID")
```

Comment:
What is your intention with `coldat` here? It doesn't seem to be saved to `mae`

Reply:
my fault. Fixing it.